src/fmu/sumo/uploader/_sumofile.py

```diff
@@ -349,5 +349,26 @@ def upload_to_sumo(self, sumo_parent_id, sumo_connection):
             self._delete_metadata(sumo_connection, self.sumo_object_id)
         else:
             result["status"] = "ok"
+            file_path = self.path
+            metadatafile_path = _path_to_yaml_path(file_path)
+            if sumo_mode == "MOVE":
```

Comment:
As above, prefer lower case. Possibly also convert input to lowercase to sanitize (be case insensitive)

Reply:
Done